use the default radondb-mysql80-template,and create cluster ,then the mysql container start fail

**Describe the problem**

<!-- Please describe the issue you observed, and any steps we can take to reproduce it: -->

<!-- If applicable, add screenshots to help explain your problem. -->
use the default radondb-mysql80-template,and create cluster ,then the mysql container start fail,the error info as follows:

```
2022-10-26T15:11:27.455956+08:00 0 [ERROR] [MY-013129] [Server] A message intended for a client cannot be sent there as no client-session is attached. Therefore, we're sending the information to the error-log instead: MY-001146 - Table 'mysql.component' doesn't exist

 2022-10-26T15:11:27.456187+08:00 0 [Warning] [MY-013129] [Server] A message intended for a client cannot be sent there as no client-session is attached. Therefore, we're sending the information to the error-log instead: MY-003543 - The mysql.component table is missing or has an incorrect definition.

 2022-10-26T15:11:27.456501+08:00 0 [ERROR] [MY-000067] [Server] unknown variable 'query-cache-size=0'.

 2022-10-26T15:11:27.457138+08:00 0 [ERROR] [MY-010119] [Server] Aborting

 2022-10-26T15:11:28.830301+08:00 0 [System] [MY-010910] [Server] /usr/sbin/mysqld: Shutdown complete (mysqld 8.0.25-15)  Percona Server (GPL), Release 15, Revision a558ec2.
```
the radondb-mysql80-template:
![image](https://user-images.githubusercontent.com/86407305/197961264-b78f7970-4766-428e-9346-593e30926f76.png)

**To Reproduce**

<!-- What did you do? Describe in your own words. -->

<!-- If possible, provide steps to reproduce the behavior: -->

1.helm install demo .
2.modify mysql_v1alpha1_mysqlcluster_mysql8.yaml,add  the infomation mysqlConfTemplate: "radondb-mysql80-template"
3.use kubectl apply to create cluter

**Expected behavior**

<!-- A clear and concise description of what you expected to happen. -->

**Environment:**

<!--
e.g.
- Kubernetes version: v1.19.5
- Helm version: v3.4.2
- RadonDB MySQL version: https://github.com/radondb/radondb-mysql-kubernetes/commit/56896cd119d9a6017e7f15f1dc0b37b83a720278
-->

- RadonDB MySQL version:v2.3.0-alpha.1

--- a/charts/mysql-operator/templates/mysql80_template.yaml
+++ b/charts/mysql-operator/templates/mysql80_template.yaml
@@ -9,13 +9,9 @@ metadata:
 data:
   my.cnf: |
     [mysqld]
-    expire-logs-days                = 7
-    master_info_repository          = TABLE
-    query-cache-size                = 0
-    query-cache-type                = 0
-    relay_log_info_repository       = TABLE
-    slave_rows_search_algorithms    = INDEX_SCAN,HASH_SCAN
-    sql-mode                        = STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_VALUE_ON_ZERO,NO_ENGINE_SUBSTITUTION,NO_ZERO_DATE,NO_ZERO_IN_DATE,ONLY_FULL_GROUP_BY
+    binlog_expire_logs_seconds      = 604800
+    default-authentication-plugin   = mysql_native_password
+    sql-mode                        = STRICT_TRANS_TABLES,NO_ENGINE_SUBSTITUTION
     binlog_format                   = row
     default-time-zone               = +08:00
     enforce-gtid-consistency        = ON
@@ -68,8 +64,6 @@ data:
     performance_schema              = 1
     slave_parallel_workers          = 8
     slave_pending_jobs_size_max     = 1073741824
-    binlog_expire_logs_seconds      = 604800
-    default-authentication-plugin   = mysql_native_password
     federated
     skip-host-cache
     skip-name-resolve
@@ -79,7 +73,6 @@ data:
     !includedir /etc/mysql/conf.d 
   plugin.cnf: |
     [mysqld]
-    plugin-load                                     = "semisync_master.so;semisync_slave.so;audit_log.so;connection_control.so"
     audit_log_buffer_size                           = 16M
     audit_log_exclude_accounts                      = "root@localhost,root@127.0.0.1,radondb_repl@%,radondb_metrics@%"
     audit_log_file                                  = /var/log/mysql/mysql-audit.log
@@ -90,6 +83,7 @@ data:
     connection_control_failed_connections_threshold = 3
     connection_control_max_connection_delay         = 2147483647
     connection_control_min_connection_delay         = 1000
+    plugin-load                                     = "semisync_master.so;semisync_slave.so;audit_log.so;connection_control.so"
     rpl_semi_sync_master_enabled                    = OFF
     rpl_semi_sync_master_timeout                    = 1000000000000000000
     rpl_semi_sync_master_wait_no_slave              = ON

